zipkin-server/README.md

```diff
@@ -302,11 +302,12 @@ Environment Variable | Property | Description
 
 
 ### ActiveMQ Collector
-The [ActiveMQ Collector](../zipkin-collector/activemq) is enabled when `ACTIVEMQ_URL` is set to a v5.x broker. The following
-settings apply in this case.
+The [ActiveMQ Collector](../zipkin-collector/activemq) is enabled when `ACTIVEMQ_URL` is set to a v5.x broker and 
+`COLLECTOR_ACTIVEMQ_ENABLED` is true. The following settings apply in this case.
 
 Environment Variable | Property | Description
 --- | --- | ---
+`COLLECTOR_ACTIVEMQ_ENABLED` | `zipkin.collector.activemq.enabled` | `true` enables the ActiveMQ collector. Defaults to `false`.
```

Comment:
hmmm I don't think we want to do this :) this means you have to set 2 properties where before only 1.

I think we want something like XXX will start collecting from blah, unless XXXX_ENABLED is set to false.

Reply:
Rolled back :) and added `enabled` to conditional evaluation.

All collectors will be "enabled=true", but actually enabled as before when url/bootstrap-servers is defined. Then if `enabled=false`, it will be disabled.